src/React.Core/IReactIdGenerator.cs

```diff
@@ -7,18 +7,14 @@
  *  of patent rights can be found in the PATENTS file in the same directory.
  */
 
-using System;
-using Xunit;
-
-namespace React.Tests.Core
+namespace React
 {
-	public class GuidExtensionsTests
+	public interface IReactIdGenerator
```

Comment:
Ugh, sorry. Missed this one. There is no XML comment for this type which is generating a warning at build time.